src/test/bundle_test.ts

```diff
@@ -391,4 +394,108 @@ suite('BuildBundler', () => {
     // We should not have the inlined file in the output.
     assert.isNotOk(getFile('framework.html'));
   });
+
+  suite('options', () => {
+
+    const projectOptions = {
+      root: 'test-fixtures/test-project',
+      entrypoint: 'index.html',
+      fragments: ['shell.html'],
+      sources: ['index.html', 'shell.html', 'source-dir/my-app.html']
+    };
+
+    test('excludes: a file', async() => {
```

Comment:
we haven't decided on this as a team so totally up to you, but I prefer test description to explain the intention / what's being tested. 

```diff
-test('excludes: a file', async() => {
+test('excludes a file from bundling when options.excludes includes that file path', async() => {
```

(to prove this point a bit, I actually had to re-read the contents of the test to understand what was being tested, since I incorrectly assumed an excluded file wouldn't be returned by `getFile()`)